specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/1.2-preview/iotcentral.json

```diff
@@ -0,0 +1,5641 @@
+    },
+    "DestinationId": {
+      "in": "path",
+      "name": "destinationId",
+      "description": "Unique ID for the destination.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "DeviceGroupId": {
+      "in": "path",
+      "name": "deviceGroupId",
+      "description": "Unique ID for the device group.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "DeviceId": {
+      "in": "path",
+      "name": "deviceId",
+      "description": "Unique ID of the device.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "DeviceTemplateId": {
+      "in": "path",
+      "name": "deviceTemplateId",
+      "description": "Unique [Digital Twin Model Identifier](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md#digital-twin-model-identifier) of the device template.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "ExportId": {
+      "in": "path",
+      "name": "exportId",
+      "description": "Unique ID for the export.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "Filter": {
+      "in": "query",
+      "name": "$filter",
+      "description": "Filter string for results.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": false
+    },
+    "JobId": {
+      "in": "path",
+      "name": "jobId",
+      "description": "Unique ID of the job.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "ModuleName": {
+      "in": "path",
+      "name": "moduleName",
+      "description": "Name of the device module.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "OrderBy": {
+      "in": "query",
+      "name": "$orderby",
+      "description": "The order by string for results.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": false
+    },
+    "RelationshipId": {
+      "in": "path",
+      "name": "relationshipId",
+      "description": "Unique ID of a relationship between devices.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "Subdomain": {
+      "in": "path",
+      "name": "subdomain",
+      "description": "The application subdomain.",
+      "x-ms-parameter-location": "client",
+      "type": "string",
+      "required": true
+    },
+    "TelemetryName": {
+      "in": "path",
+      "name": "telemetryName",
+      "description": "Name of this device telemetry.",
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "required": true
+    },
+    "Top": {
+      "in": "query",
+      "name": "$top",
+      "description": "Page size of returned results.",
```

Comment:
If this parameter specifies the page size of the results then it MUST NOT be called "top" (or "$top").  "top" should only be used for specifying "the maximum number of resources to return from the collection" [[ref](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#query-options)].

Please change the parameter name to "maxpagesize" or correct the description.

Reply:
Agreed, will update to "the maximum number of resources to return from the collection",  $top, $filter, $orderby is what we start from OData standard.